app/services/feeds/notification.rb

```diff
@@ -1,14 +1,14 @@
 module Feeds
   class Notification
-    def initialize(updated_at_from = nil, updated_at_to = nil)
-      @updated_at_from = updated_at_from || Time.zone.yesterday.beginning_of_day
-      @updated_at_to = updated_at_to || Time.zone.yesterday.end_of_day
+    def initialize(created_at_from = nil, created_at_to = nil)
+      @created_at_from = created_at_from || Time.zone.yesterday.beginning_of_day
+      @created_at_to = created_at_to || Time.zone.yesterday.end_of_day
 
       @feed = []
     end
 
     def call
-      ::Notification.includes(:notification_type).updated_at_range(@updated_at_from, @updated_at_to).find_each do |notification|
+      ::Notification.created_at_range(@created_at_from, @created_at_to).find_each do |notification|
```

Comment:
just wondering - do you need `.includes(:notification_type)` ?

Reply:
We don't. It was not needed in the first place. It was added because we originally didn't know that the id of the notification type was actually revealing by itself.